modules/configuration/ajax/process.php

```diff
@@ -59,7 +80,40 @@
     }
 }
 
+/**
+ * Query DB for config settings that correspond to filepaths.
+ *
+ * @return array IDs corresponding to path settings.
+ */
+function getPathIDs()
+{
+    $ids = array();
+    /* Query adapated from:
+     * https://github.com/aces/Loris/wiki/Project-Customization
+     */
+    $query  = "SELECT c.ID FROM Config c "
+        . "LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID) "
+        . "JOIN ConfigSettings csp ON (cs.Parent = csp.ID) "
+        . "WHERE cs.DataType = 'web_path';";
+    $result = \Database::singleton()->pselect($query, array());
+    foreach ($result as $key => $value) {
+        $ids[] = intval($value['ID']);
+    }
+    return $ids;
+}
 
-exit();
+/**
+ * Return a json-encoded error message and response code when an error occurs
+ *
+ * @param int    $code A valid HTTP Response Code.
+ * @param string $msg  Error text to display.
+ *
+ * @return void
+ */
+function displayError(int $code, string $msg): void
+{
+    // Display error message and return
+    http_response_code($code);
+    echo "ERROR: $msg";
```

Comment:
Hi @johnsaigle I'm not sure if it should be addressed in this PR but we're now using new \LORIS\Http\Response()) for the ajax.

Reply:
Yeah I think that depends on a larger refactoring of the module which isn't what this PR aims to do. it's a patch over the old way of doing things. Config eventually needs to be reworked.